File: wetrot-backend-java/src/main/java/io/wesquad/wetrot/model/Trot.java
package io.wesquad.wetrot.model;

import lombok.*;
import org.springframework.data.mongodb.core.mapping.Document;

import java.util.UUID;

@Data
@AllArgsConstructor
@NoArgsConstructor
@Builder
@ToString
@Document(value = "trot")
public class Trot {
    private UUID uid;
    private String name;
    private int currentBatteryLevel;
    private int batteryCapacity;
    private String localization;
    private String brand;
    private int usury;
    private int wearLife;
}

File: wetrot-backend-nestjs/test-files/service/trot.ts
import { TrotDto } from '../../src/trot/model/trotDto';
import { ITrot } from 'src/trot/model/trot.i';

export const trotDto = new TrotDto();
trotDto.brand = 'brand';
trotDto.name = 'name';
trotDto.batteryCapacity = 5500;
trotDto.currentBatteryLevel = 2000;
trotDto.usury = 0;
trotDto.wearLife = 990;

export const itrot: ITrot = {
    id: 1,
    brand: 'brand',
    name: 'name',
    batteryCapacity: 5500,
    localization: '48.8534;2.3488',
    currentBatteryLevel: 2000,
    usury: 0,
    wearLife: 990,
};
